salt_sproxy/parsers.py

```diff
@@ -239,13 +239,21 @@ def _mixin_setup(self):
             help='Show the devices expected to match the target.',
         )
         self.add_option(
-            '--sync-modules',
-            dest='sync_modules',
+            '--sync-grains',
+            dest='sync_grains',
```

Comment:
I am wondering if `--sync-modules --sync-grains --sync-pillars` etc is better than `--sync m,g,p` or `--sync mods,grains,pillars` whatsoever.

Reply:
That might be a good idea. I think I incline to add these one individually, i.e., `--sync-grains` etc. together with that alternative you suggested.

Reply:
That's just an idea, surely not for _this_ PR. :wink: